tests/test_order_Steinhardt.py

```diff
@@ -278,7 +282,8 @@ def test_compute_twice_norm(self):
 
         npt.assert_array_almost_equal(first_result, second_result)
 
-    def test_rotational_invariance(self):
+    @pytest.mark.parametrize("i", range(10))
+    def test_rotational_invariance(self, i):
```

Comment:
Let's use the name `seed` instead of `i`. It isn't clear how `i` is used, but parametrizing over multiple seeds is easier to understand.
```suggestion
    @pytest.mark.parametrize("seed", range(10))
    def test_rotational_invariance(self, seed):
```